lib/sidekiq-scheduler/version.rb

```diff
@@ -1,3 +1,3 @@
 module SidekiqScheduler
-  VERSION = "5.0.6"
+  VERSION = "5.0.7"
```

Comment:
Please let me know if you would prefer a different version number, like 5.1, etc.